src/components/ExampleForm.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { addItem } from "../js/actions/index";
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    addItem: (data) => dispatch(addItem(data))
+  };
+};
+
+class ConnectedExampleForm extends Component {
```

Comment:
Can we switch this to a functional component and use hooks for consistency of structure?

Reply:
@jarrodchung-dev